src/sources.cr

```diff
@@ -36,11 +62,12 @@ class LiveSource < Source
 
       if parsed_message.has_key? "value"
         @last_message = parsed_message["value"]
-        @tx.send({@ident, @last_message.as_h})
       elsif parsed_message.has_key? "delta"
         apply_patch(parsed_message["delta"])
-        @tx.send({@ident, @last_message.as_h})
       end
+
+      @current_data.from_stream @last_message
```

Comment:
Will this resend to clients on every message, even if the patch fails?

Reply:
oh, good point!